pkg/config/controller/config.go

```diff
@@ -91,3 +93,9 @@ type IPsecCSRSignerConfig struct {
 	// Defaults to true.
 	AutoApprove *bool `yaml:"autoApprove,omitempty"`
 }
+
+type AuthenticationOption struct {
+	// ClientCA is the file path of the certificate bundle for all the signers that is recognized for incoming
```

Comment:
Questions - are we able to support cert change without restarting antrea-controller? How about other certs? Do we use Secret or file path to specify a cert?

Reply:
Here we expects to use a file path to specify a cert. We can generate a file by mounting a Secret in the Pod's configuration.

Reply:
Could you answer my previous questions:
> Questions - are we able to support cert change without restarting antrea-controller? How about other certs?

I know we support rotation of antrea-controller-tls certs, as described here: https://github.com/antrea-io/antrea/blob/main/docs/securing-control-plane.md.

Reply:
Yes, antrea would recognize the certificate changes automatically without a changes. Here we are using dynamic CAContentProvider (https://github.com/kubernetes/apiserver/blob/v0.24.0/pkg/server/dynamiccertificates/dynamic_cafile_content.go#L76) to provide the CA content in the related authentication functions for apiserver, which has a watcher on the changes of the specified CAFile. 

So as long as the Secret is changed, the content of the mounted file changes, then the provider would reload content.

Reply:
Ok. Thanks!